b2b-sample/guardio-insurance-sample-application/util/util/orgUtil/orgUtil.js

```diff
@@ -57,6 +57,16 @@ function getOrgIdfromRouterQuery(routerQuery) {
     return undefined;
 }
 
+function getOrgIdFromQuery(query) {
```

Comment:
Can we simplify this method?

Suggestion
```
function getOrgIdFromQuery(query) {
    const org = config.SAMPLE_ORGS.find(org => org.routerQuery === query);
    return org ? org.id : null;
}
```

Reply:
Fixed with
https://github.com/Achintha444/samples-is/commit/869e218ddd640521a7a5629b4ba6bf12e1e1afea#diff-54e1f2d15b1f45bc469f601fb94904610dd9f62dea8bcc1a3f2a64a73686ff29